packages/uploadthing/test/sdk.test.ts

```diff
@@ -75,111 +48,79 @@ describe("uploadFiles", () => {
   });
 
   it("returns array if array is passed", async ({ db }) => {
-    const utapi = new UTApi({ apiKey: "sk_foo" });
+    const utapi = new UTApi({ token: testToken.encoded });
     const result = await utapi.uploadFiles([fooFile]);
     expectTypeOf<UploadFileResult[]>(result);
     expect(Array.isArray(result)).toBe(true);
   });
 
   it("returns single object if no array is passed", async ({ db }) => {
-    const utapi = new UTApi({ apiKey: "sk_foo" });
+    const utapi = new UTApi({ token: testToken.encoded });
     const result = await utapi.uploadFiles(fooFile);
     expectTypeOf<UploadFileResult>(result);
     expect(Array.isArray(result)).toBe(false);
   });
 
   it("accepts UTFile", async ({ db }) => {
-    const utapi = new UTApi({ apiKey: "sk_foo" });
+    const utapi = new UTApi({ token: testToken.encoded });
     const file = new UTFile(["foo"], "foo.txt");
     await utapi.uploadFiles(file);
     expect(file.type).toBe("text/plain");
 
     expect(requestSpy).toHaveBeenCalledWith(
-      "https://uploadthing.com/api/uploadFiles",
-      expect.objectContaining({}),
+      expect.stringContaining(`x-ut-file-name=foo.txt`),
```

Comment:
maybe make a `expect.urlMatching({ host, search ... })` helper to assert more stuff